types/block.go

```diff
@@ -1132,11 +1145,10 @@ func (data *EvidenceData) FromProto(eviData *tmproto.EvidenceData) error {
 			return err
 		}
 		eviBzs[i] = evi
+		data.byteSize += int64(eviData.Evidence[i].Size())
```

Comment:
not sure if this is correct, this accounts of the byte size of all the individual evidence but not as though it was wrapped in `message EvidenceData`

Reply:
Yeah it's also missing the size of the hash of `EvidenceData`

Reply:
should be just data.byteSize = eviData.Size() at the end